docs/administering/NOTIFYING_LISTSERV.md

```diff
@@ -0,0 +1,37 @@
+# Notifying the listserv of updates
+
+We have a google group set up to let funds know about DARIA updates and other important matters. Generally we email them as a courtesy when deploying updates, so they're aware of new features, security patches, etc. This is a guide to assembling the notification email.
+
+## You can copy this part
+
+> Hi, we're deploying the following updates to DARIA shortly:
+
+### And then we get the bullets of specific updates
+
+* Go to the Heroku dashboard to figure out when the last deploy to production was
```

Comment:
Can we teach heroku to tag this in GH? :thinking:

Reply:
Almost definitely!